test/functional/page_objects/settings_page.ts

```diff
@@ -681,9 +683,44 @@ export function SettingsPageProvider({ getService, getPageObjects }: FtrProvider
       });
     }
 
+    async getSavedObjectElementsInTable() {
+      const rows = await testSubjects.findAll('~savedObjectsTableRow');
+      return mapAsync(rows, async row => {
+        const checkbox = await row.findByCssSelector('[data-test-subj*="checkboxSelectRow"]');
+        // return the object type aria-label="index patterns"
+        const objectType = await row.findByCssSelector('[data-test-subj="objectType"]');
```

Comment:
with the latest master: `const objectType = await row.findByTestSubject('objectType');`